cypress/selectorHelpers/index.jsx

```diff
@@ -0,0 +1,24 @@
+const selectDataHook = hook => `[data-test="${hook}"]`; // writes to component
```

Comment:
This is the string CSS selector you'll plug into your Enzyme .find() or your Cypress.get(...)